.vscode/launch.json

```diff
@@ -116,6 +116,24 @@
         "--webpack-config",
         "webpack.config.test.js"
       ]
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Aztec 2D Tests - ts-node",
```

Comment:
Is this faster than webpack-mocha?

Reply:
I struggled to debug with mocha-webpack, either the source maps didn't match the typescript sources or it just didn't stop at the break points. mocha-webpack seems not maintained anymore since longer time. But it could be my unknowing in this topic.
This setup did just work fine for.

Reply:
Nice to know that, it was just the opposite for me.